web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -62,4 +63,55 @@ describe("<SidebarCardsList />", () => {
     fireEvent.click(screen.getByTestId("hidden-opportunity-header"));
     expect(mockAnalytics.event.for_you_card_unhide_button.click.unhide_cards).toHaveBeenCalledTimes(1);
   });
+
+  describe("empty state messaging", () => {
+    it("displays when the For You section is empty", () => {
+      render(
+        <SidebarCardsList
+          topCards={[]}
+          bottomCards={[]}
+          fundings={[]}
+          hiddenFundings={[]}
+          certifications={[]}
+          hiddenCertifications={[]}
+          displayFundings={false}
+          displayCertifications={false}
+        />
```

Comment:
[sand] Every test in this function has roughly the same render function. You could reduce each test by 10 lines by creating a global render function that takes the desired props for a given test. Then you could call something like:
```
renderComponent({displayFundings: false});
```

This would make the test more readable, and you'd have the added benefit of being able to clearly see what parameters are being tested in the specific test.